src/Game.js

```diff
@@ -8,32 +8,118 @@ class Game {
     this.playersList = players;
     this.generatedPlayers;
     this.allCategories = this.data.categories;
-    this.currentCategories = [];
+    this.currentCategories;
     this.clues;
     this.winner;
-    this.roundCounter;
+    this.roundCounter = 1;
     this.currentRound;
+    this.startGame();
+
```

Comment:
This is a lottttt of instance properties. And I'm not entirely sure the distinction between some of them.  PlayersList vs. generatedPlayers? allCategories vs. currentCategories? roundCounter vs. currentRound?  Be careful of turning your game into a "God Class" like the readings mentioned yesterday. Some of these should probably belong elsewhere, and I would question if they are all even necessary.